README.md

```diff
@@ -250,20 +267,22 @@ The **summary.tsv** output file generated by `staramr` contains the following co
 * __Genotype__: The AMR genotype of the isolate.
 * __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) for the isolate.
 * __Plasmid__: Plasmid types that were found for the isolate.
+* __Scheme__: The organism that `MLST` has identified.
```

Comment:
Can you change to **The MLST scheme used.**

Reply:
Fixed in my next commit